docs/k0s-reset.md

```diff
@@ -0,0 +1,35 @@
+# k0s reset
+
+k0s Command-Line interface now allow users to entirely remove all k0s related files from the host.
+
+The reset command operates under assumption that k0s is running as service on the host. So first we need to stop the service:
```

Comment:
```suggestion
The reset command operates under assumption that k0s is running as service on the host. and to prevent accidental triggering, it will not run if the k0s service is up. So first, we need to stop the service:
```

Reply:
`as a service`